psi_cardinality/c/internal_utils.h

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef PSI_CARDINALITY_INTERNAL_UTILS_H_
+#define PSI_CARDINALITY_INTERNAL_UTILS_H_
+
+#include "util/statusor.h"
+
+int generate_error(private_join_and_compute::Status status, char **error_out);
```

Comment:
Since this is only included from C++ files, let's put it in an internal namespace, e.g., `psi_cardinality::c_bindings_internal`